src/devices/bus/a2bus/uthernet.cpp

```diff
@@ -0,0 +1,77 @@
+// license:BSD-3-Clause
+// copyright-holders:R. Belmont
+/*********************************************************************
+
+    uthernet.cpp
+
+    Apple II Uthernet Card
+
+*********************************************************************/
+
+#include "emu.h"
+#include "emuopts.h"
+#include "uthernet.h"
```

Comment:
The module header should be the first thing included after the prefix header.  It should start like this:
```
#include "emu.h"
#include "uthernet.h"

#include whatever other headers
```

If it doesn’t work like that, the header needs to be fixed.